integration/integration_test.go

```diff
@@ -221,6 +230,71 @@ func call[Resp any](module, verb string, req obj, onResponse func(t testing.TB,
 	}
 }
 
+func setUpKotlinModuleDb(dir string) assertion {
```

Comment:
This should just be `setupModuleDb` as it's not Kotlin specific.

Reply:
updated - right now this scaffolds the kotlin `DbTest.kt` file, but we can always follow up with a change to write language-appropriate content once Go is ready